packages/firebase/index.d.ts

```diff
@@ -6060,6 +6061,20 @@ declare namespace firebase.firestore {
      */
     app: firebase.app.App;
 
+    /**
+     * Clears the persistent storage.
+     *
+     * Must be called after the app is shutdown or before other methods (other
+     * than settings()) on startup. If the client is still running, an exception
+     * `code` will be thrown.
+     *
+     *   * failed-precondition: The client is still running in a browser tab.
```

Comment:
Use bulleted lists only when you have more than one item in the list. I suggest:

> ..., an exception with a code of `failed-precondition` will be thrown.